gcloud/bigquery/client.py

```diff
@@ -96,3 +97,20 @@ def dataset(self, name):
         :returns: a new ``Dataset`` instance
         """
         return Dataset(name, client=self)
+
+    def load_from_storage(self, name, destination, *source_uris):
```

Comment:
OK. The name just is odd to me. Like if I didn't read the docstring I'd never know what was being loaded.